src/components/Chatting/Chattingroom.tsx

```diff
@@ -147,31 +144,39 @@ function Chattingroom({ roomId, openAuctionInfo, lastChat }: ChatroomProps) {
       <div className={S.chatroomHeader}>
         <span className={S.chatroomName}>{openAuctionInfo?.room_name}</span>
         <div className={S.chatroomUserSection}>
-          <span className={S.user}>{`판매자: ${seller}`}</span>
-          <span className={S.user}>{`구매자: ${buyer}`}</span>
+          <span
+            className={S.user}
+          >{`구매자: ${openAuctionInfo?.buyer_name || openAuctionInfo?.buyer_id}`}</span>
+          <span
+            className={S.user}
+          >{`판매자: ${openAuctionInfo?.seller_name || openAuctionInfo?.seller_id}`}</span>
         </div>
       </div>
       <div ref={chatContainerCallbackRef} className={S.chatListWrapper}>
         {messages.map(msg => {
           return (
             <div
-              key={`${msg.created_date}+${msg.message}+${msg.sender_id}`}
-              className={msg.sender_id === user?.member_id ? S.msgBox.myMsg : S.msgBox.opponentMsg}
+              key={`${msg.created_at}+${msg.message}+${msg.sender_member_id}`}
+              className={
+                msg.sender_member_id === user?.member_id ? S.msgBox.myMsg : S.msgBox.opponentMsg
+              }
               // 이 삼항 연산 부분 리팩토링 필요
             >
               <div
                 className={
-                  msg.sender_id === user?.member_id ? S.msgWrapper.myMsg : S.msgWrapper.opponentMsg
+                  msg.sender_member_id === user?.member_id
+                    ? S.msgWrapper.myMsg
+                    : S.msgWrapper.opponentMsg
                 }
               >
                 <span className={S.msg}>{msg.message}</span>
               </div>
               <p
                 className={
-                  msg.sender_id === user?.member_id ? S.msgDate.myMsg : S.msgDate.opponentMsg
+                  msg.sender_member_id === user?.member_id ? S.msgDate.myMsg : S.msgDate.opponentMsg
                 }
               >
-                {msg.created_date}
+                {msg.created_at}
```

Comment:
요거 ISO 시간으로 나와서 formatDate() 유틸 추가 필요해 보여요!
```suggestion
                {formatDate(msg.created_at)}
```

Reply:
이런 좋은 훅이 있었군요!!!